elki-core-math/src/test/java/elki/math/statistics/dependence/mcde/MWPTestTest.java

```diff
@@ -63,4 +62,18 @@ public void testRanking() {
     assertEquals("ranks incorrect", output_duplicates_start_end.adjusted[11], 11.0, 0);
     assertEquals("ranks incorrect", output_duplicates_start_end.adjusted[12], 11.0, 0);
   }
+
+  @Test
+  public void testStatTest(){
+    int len = 10;
+    DoubleArrayAdapter adapter = DoubleArrayAdapter.STATIC;
+    MWPTest mwpTest = MWPTest.STATIC;
```

Comment:
As these are used only once, I think they should just be used below without predeclaration.

Reply:
Done